koschei/frontend/views.py

```diff
@@ -647,8 +647,9 @@ def bugreport(name):
     variables = package.srpm_nvra or abort(404)
     variables['package'] = package
     variables['collection'] = package.collection
-    variables['url'] = request.url_root.replace(request.script_root, '').rstrip('/') \
-        + url_for('package_detail', name=package.name)
+    external_url = frontend_config.get('external_url', request.host_url).rstrip('/')
```

Comment:
I don't see `request.host_url` in [flask documentation](http://flask.pocoo.org/docs/0.12/api/#flask.Request). Is it documented? I would prefer not to rely on undocumented fields.

Reply:
From flask documentation (the page you linked):
The request object is a Request subclass and provides all of the attributes Werkzeug defines plus a few Flask specific ones.

It is documented in Werkzeug documentation:
http://werkzeug.pocoo.org/docs/0.14/wrappers/#werkzeug.wrappers.BaseRequest.host_url